ext/nnstreamer/tensor_filter/tensor_filter_pytorch.cc

```diff
@@ -477,12 +482,20 @@ TorchCore::invoke (const GstTensorMemory * input, GstTensorMemory * output)
       ml_loge ("Output Tensor Information is not valid");
       return -2;
     }
+#ifdef PYTORCH_VER_ATLEAST_1_2_0
+  } else if (output_value.isList ()) {
+    c10::ArrayRef < torch::jit::IValue > output_ref_list =
+      output_value.toListRef ();
+    std::vector < torch::jit::IValue > output_list (
```

Comment:
I found that the TAOS-CI have not checked the clang format '.cc' files (https://github.com/nnstreamer/TAOS-CI/pull/632).
Anyway, you have to fix the format later.

Reply:
Applying the clang format syntax to the file produces changes beyond the scope of this PR.
I have just applied clang-format to this changes introduced in this PR. 
Extensions will have to updated with syntax in a separate PR.

Reply:
Sure, I didn't mean that you should change the format in this PR.